bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BTableType.java

```diff
@@ -56,12 +57,12 @@ public BType getKeyType() {
 
     @Override
     public <V> V getZeroValue() {
-        return null;
+        return (V) new TableValueImpl<String, V>(new BTableType(constraint));
```

Comment:
we cant have string here. key type can be any anydata type for tables.

Reply:
Fixed.